assets/js/modules/pagespeed-insights/index.legacy.js

```diff
@@ -37,7 +37,7 @@ if ( ! active || ! setupComplete ) {
 	addFilter(
 		'googlesitekit.DashboardModule',
 		'googlesitekit.PageSpeedInsights',
-		createAddToFilter( <PageSpeedInsightsCTA /> ),
+		createAddToFilter( <DashboardPageSpeedCTA /> ),
 		45
 	);
```

Comment:
Actually, now that this component is refactored, let's move this filter to the regular `index.js` too, and then remove `index.legacy.js`.

Reply:
Done.